packages/cli/internal/pkg/cli/context_status.go

```diff
@@ -54,7 +55,11 @@ func BuildContextStatusCommand() *cobra.Command {
 			if err != nil {
 				return clierror.New("context status", nil, err)
 			}
-			format.Default.Write(contextInstances)
+			if len(contextInstances) > 0 {
+				format.Default.Write(contextInstances)
```

Comment:
There is a function that is responsible for printing out collections. Will make sense to move this functionality to that function? In this case all commands that output a collection would benefit from this change.

This code could be also simplified:

```go
if len(contextInstances) == 0 {
  log.Info().Msg("There are no contexts deployed.")
}
format.Default.Write(contextInstances)
```